packages/lab/stories/tabs/tabs.doc.stories.mdx

```diff
@@ -1,23 +1,82 @@
+
+The easiest way to create a Tabbed view with all the appropriate `aria` relationships in place, is to use the `Tabs` component with `TabPanel`, rather than create a `Tabstrip` directly.
+
+<Canvas withSource="open">
+  <SaltProvider>
+    <Tabs>
+      <TabPanel label="Home">
+        <Text>Home Page</Text>
+      </TabPanel>
+      <TabPanel label="Transactions">
+        <Text>Transaction Page</Text>
+      </TabPanel>
+      <TabPanel label="Loans">
+        <Text>Loans Page</Text>
+      </TabPanel>
+      <TabPanel label="Checks">
+        <Text>Checks Page</Text>
+      </TabPanel>
+      <TabPanel label="Liquidity">
+        <Text>Liquidity Page</Text>
+      </TabPanel>
+    </Tabs>
+  </SaltProvider>
+</Canvas>
+
+This not only ensures that the correct `aria` attributes are applied, but also takes
+care of the display switching between active and non-active `TabPanel` elements.
+
+<br />
+
+Use of the `Tabs` component is recommended, but it isn't always possible. Sometimes the `Tabstrip` and associated elements will be in separate locations on the page. The Tabstrip may be housed within an `AppHeader` or Sidebar or other layout structure. In such cases, it is even more important that the correct `aria` attributes are applied to make the relationships between the various UI elements clear to screen readers. This requires a little more effort than when `Tabs` are employed. The example below illustrates how this can be done.
+
+<Canvas withSource="open">
+  <Story id="lab-tabstrip--default" />
+</Canvas>
 
-# Tabstrip
+By creating `Tab` components, we are able to assign an id value to the `ariaControls` prop. We then assign that same id value to the active `tabpanel`. We can also establish the reverse relationship - the `aria-labelledby` attribute lives on the `tabpanel`. We can assign to this the id of the corresponding `Tab` because the id values of tabs are derived from the id value we pass to the `Tabstrip` itself. In this case, we pass the value 'ts' as the `Tabstrip` id. Our tabs will have default id values of 'ts-0', 'ts-1', 'ts-2' etc.
 
-Tabs are defined either declaratively, by nesting `Tab` elements or via
-configuration - use of the `tabs` or `defaultTabs` props.
+Tabs are defined either declaratively, by nesting `Tab` elements or via configuration - use of the `tabs` or `defaultTabs` props.
 
 <br />
 
```

Comment:
This file I merged with [packages/lab/stories/tabs/tabs.stories.mdx](https://github.com/jpmorganchase/salt-ds/pull/1089/files#diff-bc9d9f1a8c05d4dd6dc9cdf95ee8db2cdf5b595ef1125c5b93fe88aeb7d856a7) (and subsequently deleted) as they contained overlapping info - whoever works on Tab/Tabstrip may want to refine and split this doc more appropriately